src/50_Samples_%26_Templates/Product.html

```diff
@@ -2,7 +2,7 @@
   "preview": "default",
   "skipValidation": "true",
   "default": "preview",
-  "experiments": ["amp-form-var-sub"]
+  "experiments": ["amp-form-var-sub", "amp-bind"]
   }--->
 <!--
 #### Introduction
```

Comment:
Add an empty comment `<!-- -->` after the introduction to make the intro print before the code. Currently the whole header + CSS is shown in the annotated source mode ... which is very long.